seed/main/trw.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<ldml>
+
+	<identity>
+
+		<version number="$new$"/>
+
+		<language type="trw"/>
+
+		<script type="Arab"/>
+
+	</identity>
+
+	<layout>
+
+		<orientation>
+
+			<characterOrder>right-to-left</characterOrder>
+
+		</orientation>
+
+	</layout>
+
+	<characters>
+
+		<exemplarCharacters draft="confirmed">[ا آ أ ب پ ت ٹ ث ج ݲ چ ڇ څ ح خ د ڈ ذ ر ڑ ز ژ ڙ س ش ص ض ط ظ ع غ ف ق ک گ ل م ن ں ہ ھ ء ی  ئ ے]</exemplarCharacters>
```

Comment:
change it to `unconfirmed` (just for now)

Reply:
errors removed